### DISCUSSION TITLE
Planned deprecations in upcoming Lagoon major releases

### DISCUSSION THREAD

**Discussion by tobybellwood** [uselagoon/lagoon#3580] [Roadmap]
Here is a placeholder issue to capture deprecations planned for the next major release of Lagoon:

* Deprecate support for Harbor attached to lagoon-core. This has been sup[erseded by the lagoon-remote integration. This 
* Removal of all "Openshift" and "Kubernetes" references in the API and database, replaced by "DeployTarget" or similar

**Comment by smlx** [comment]
I'd like to see these deprecated:
* Workloads able to be deployed as root.
* Core SSH service.
